lib/util/http/github.ts

```diff
@@ -152,8 +152,14 @@ function constructAcceptString(input?: any): string {
 }
 
 export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
-  constructor(options?: GithubHttpOptions) {
-    super(PLATFORM_TYPE_GITHUB, options);
+  constructor({
+    hostType = PLATFORM_TYPE_GITHUB,
+    options,
+  }: {
+    hostType?: string;
+    options?: GithubHttpOptions;
+  }) {
+    super(hostType, options);
```

Comment:
Use optional second parameter instead, so most code can fallback to exiting working `hostType`